lib/rake/build.rake

```diff
@@ -21,6 +21,14 @@ namespace :build do
     end
   end
 
+  desc 'Builds broken link checker.'
```

Comment:
Should desc be something like "builds a set of independent tools" and then have a comment above brokenLinkChecker that it's building this particular tool (i.e. presumably there may eventually be other tools we expect to live in here too?)